content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,68 @@
+---
+authors: ["veronica-bolognesi"]
+comments: true
+date: "2020-09-10"
+draft: false
+share: true
+categories: [English, Agile, Scrum, Kanban]
+title: "An Agile workflow"
+
+languageCode: "en-EN"
+type: "post"
+---
+On sept 9 the 2020 edition of PHPDay was held in a remote fashion https://2020.phpday.it/talks.html.
+
+Having experienced the 2019 edition (indeed my first one, despite being a PHP developer from eons!) I should say that it was a little difficult to maintain the concentration for such a long event, but the effort was well worth of, given the level of both lecturers and talks.
+
+I would like to thanks the folks at Grups https://www.grusp.org/ for their passionate work. Try to sustain them!
+
+What follows is a mere recap of the event, with my little bit of comments. 
+
+
+**Things I Wish I Knew When I Started Playing Developer by _Gary Hockin_**
+
+A trip into developers' mindset from a ludic (and hence, engaging) viewpoint. Gary is a natural born speaker ;) and this talk deserves to be seen by anyone interested in programming, no matter their background and interests, at least once. 
+
+**What's new in PHP 8 by _Derick Rethans_**
+
+Personally, I did not like the style of the presentation (not the theme of slides, but how they got presented! :D) but the information provided is very important and so you should take a look here: https://derickrethans.nl/talks/php-phpday20 
+
+**Symfony 5, the new bit by _Andreas Hucks_**
+
+A deep dive into Symfony 5 features and enhancements. Definitely to understand in order to evolve our software: we won't take dead ends if we know where the developers of our framework of choice are heading. Right?
```

Comment:
I don't understand the second sentence here.. Can you clarify it a bit?

Reply:
```suggestion
A deep dive into Symfony 5 features and enhancements. Knowing where the framework is heading will make us take better-informed decisions in our day to day work.
```

Reply:
@Jean85 Do you find it more understandable in this way?

Reply:
Concerning 

> I would suggest to follow a pattern that we used in previous articles like this one

Since we won't publish today I will take your suggestion and re-work the layout in the weekend and will also make Hugo work on my local env :)

Reply:
Great! Seems good to me!